DigitalLearningSolutions.Data/Services/CentresService.cs

```diff
@@ -48,5 +50,16 @@ FROM Centres
 
             return name;
         }
+
+        public IEnumerable<(int, string)> GetActiveCentres()
+        {
+            var centres = connection.Query<(int, string)>(
+                @"SELECT CentreID, CentreName
+                        FROM Centres
+                        WHERE Active = 1
+                        ORDER BY CentreName"

```

Comment:
Should we make it clear in the method name that these will be alphabetically ordered? Alternatively, we could do the ordering in the code so this could be reused for other functionality too, but I'm struggling to think of other places that might need all of the active centres. 🤔

Reply:
Yeah, my logic was that ordering by id would be arbitrary and not very useful, especially for such a long list, whereas ordering alphabetically would be more intuitive.

Reply:
Yeah, I agree with this so maybe just make it clear that we use ordering in the name of the method? 👍